types/custom-types.d.ts

```diff
@@ -0,0 +1,125 @@
+import { Descendant, BaseEditor } from 'slate';
```

Comment:
Why do you want to move this file into the types folder, and should the file in the root directory be deleted?

Reply:
I moved the file because it seems cleaner to me to put it in a dedicated folder.